src/plugins/intel_cpu/src/nodes/kernels/rdft_kernel.hpp

```diff
@@ -90,6 +96,12 @@ struct jit_dft_kernel_f32 : public jit_dft_kernel, public jit_generator {
         Xbyak::Reg64 signal_size = r11;
         Xbyak::Reg64 output_start = r12;
         Xbyak::Reg64 output_end = r13;
+
+        std::vector<int> perm_low_values;
+        std::vector<int> perm_high_values;
```

Comment:
minor: in general it is recommended to insert such a small constants inside the kernel itself. Otherwise it makes harder to serialize such kernels, because it forces us to keep jit_dft_kernel_f32 alive during execution